Mirror/Tests/Mirror.Tests.csproj

```diff
@@ -48,11 +48,23 @@
     <Reference Include="UnityEngine">
       <HintPath>..\lib\UnityEngine.dll</HintPath>
     </Reference>
+    <Reference Include="Castle.Core">
+      <HintPath>..\packages\Castle.Core.4.2.0\lib\net45\Castle.Core.dll</HintPath>
+    </Reference>
+    <Reference Include="System.Configuration" />
+    <Reference Include="System.Threading.Tasks.Extensions">
+      <HintPath>..\packages\System.Threading.Tasks.Extensions.4.3.0\lib\portable-net45+win8+wp8+wpa81\System.Threading.Tasks.Extensions.dll</HintPath>
+    </Reference>
+    <Reference Include="NSubstitute">
+      <HintPath>..\packages\NSubstitute.3.1.0\lib\net46\NSubstitute.dll</HintPath>

```

Comment:
why do we need caste.core and nsubstitue now?

Reply:
NSubstitute is a mocking framework,  I need it in order to test synclists (see explanation below)

caste.core is just a dependency of NSubstitute.  When I added NSubstitute, caste.core got added automatically.  I don't use it directly.

Neither one of these binaries are added to the builds. They are used to run the unit tests, but discarded afterwards.